Makefile

```diff
@@ -23,11 +26,14 @@ install-tools:
 	hack/install-kubebuilder-tools.sh
 
 test: build install-tools
-	hack/test-go.sh
+	hack/test-go.sh -u $(USE_STATIC_CHECK)
 
 kind:
 	hack/e2e-setup-kind-cluster.sh -n $(COMPUTE_NODES)
 
+upstream-test: build
+	hack/test-go.sh -u $(false)
```

Comment:
since it doesn't read the value of what `-u value` is -- I think you can remove this here with the current logic order.